data/coalition/coalition ships.txt

```diff
@@ -695,6 +1150,175 @@ ship "Heliarch Punisher" "Heliarch Punisher (Scrappy)"
 		"Hyperdrive"
 
 
+ship "Heliarch Pursuer"
+	sprite "ship/heliarch pursuer"
+	thumbnail "thumbnail/heliarch pursuer"
+	attributes
+		category "Fighter"
+		licenses
+			Heliarch
+		"cost" 1029000
+		"shields" 2200
+		"hull" 1600
+		"required crew" 1
+		"bunks" 1
+		"mass" 31
+		"drag" 0.8
+		"heat dissipation" 0.99
+		"outfit space" 105
+		"weapon capacity" 19
+		"engine capacity" 36
+		weapon
+			"blast radius" 15
+			"shield damage" 150
+			"hull damage" 200
+			"hit force" 350
+	outfits
+		"Ion Rain Gun"
+		
+		"Large Collector Module"
+		"Small Collector Module"
+		"Small Battery Module" 3
+		"Small Shield Module"
+		"Small Repair Module"
+		
+		"Small Steering Module" 2
+		"Small Thrust Module" 2
+	
+	engine 0 34.5
+	engine 7 33.5
+	engine -7 33.5
+	gun 0 -35.5
+	explode "tiny explosion" 20
+	explode "small explosion" 10
+	description `Occasionally, when a Heliarch cadet shows promising results in dogfighting simulations, they are allowed to fly a ship of their own, that being the Pursuer.`
+	description `	Though certainly not as durable as the other fighters in Coalition space, this Kimek design is cheaper to produce, and is capable of outrunning any other Coalition ship.`
+
+ship "Heliarch Pursuer" "Heliarch Pursuer (Scrappy)"
+	outfits
+		"Bombardment Cannon"
+		
+		"Large Collector Module"
+		"Small Collector Module" 3
+		"Small Battery Module"
+		"Small Shield Module"
+		"Small Repair Module"
+		
+		"Small Thrust Module" 2
+		"Small Steering Module" 2
+
+
+ship "Heliarch Rover"
+	sprite "ship/heliarch rover"
+	thumbnail "thumbnail/heliarch rover"
+	attributes
+		category "Fighter"
+		licenses
+			Heliarch
+		"cost" 1238000
+		"shields" 3300
+		"hull" 3100
+		"required crew" 1
+		"bunks" 1
+		"mass" 37
+		"drag" 1.19
+		"heat dissipation" .96
+		"outfit space" 125
+		"weapon capacity" 24
+		"engine capacity" 32
+		weapon
+			"blast radius" 15
+			"shield damage" 150
+			"hull damage" 200
+			"hit force" 350
+	outfits
+		"Bombardment Cannon" 2
+		
+		"Large Collector Module"
+		"Large Battery Module"
+		"Small Shield Module"
+		"Small Repair Module" 3
+		
+		"Small Thrust Module" 2
+		"Small Steering Module" 2
+	
+	engine 7 28
+	engine -7 28
+	gun 5 -30
+	gun -5 -30
+	explode "tiny explosion" 25
+	explode "small explosion" 15
+	description `The very first Fighter the Heliarchs designed, Rovers are well-rounded ships that the Heliarchs employ regularly due to their reliability in any situation.`
```

Comment:
```suggestion
	description `The very first fighter they designed, Rovers are well-rounded ships that the Heliarchs employ regularly due to their reliability in any situation.`
```
is `Fighter` supposed to be capitalized? I'm unsure. Also, double `the Heliarchs`.